src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -149,7 +152,8 @@ export default Vue.extend({
     hasLoadDataError: false,
     hasSaveDesignError: false,
     hasSaveDesignSuccess: false,
-    savedDesignName: null
+    savedDesignName: null,
+    showDiffFVAScore: false
```

Comment:
Right now, the state isn't really tied properly to the toggle button. If you toggle it, switch to another card, then move back, the toggle switches position, but the map doesn't change.

I think it's better to define this on each card rather than in this parent component, and access `card.showDiffFVAScore` from Escher. Then the InteractiveMap component can be agnostic about this flag.

Reply:
Gotcha! Done.